CONTRIBUTING.MD

```diff
@@ -119,31 +121,30 @@ Unit tests do not require a Databricks account. Please confirm that your pull re
 tox -e unit
 ```
 
-## Functional & Integration Tests
+## Functional Tests
```

Comment:
Also updated this section since we don't have integration tests anymore, and we don't test with a non-UC SQL warehouse.